h/security/policy/helpers.py

```diff
@@ -16,19 +16,29 @@ def is_api_request(request) -> bool:
 class AuthTicketCookieHelper:
     def identity(
         self, cookie: SignedCookieProfile, request: Request
-    ) -> Identity | None:
+    ) -> tuple[Identity, AuthTicket] | tuple[None, None]:
```

Comment:
Because of the "temporary" / "transitional" cookie-issuing code it's necessary for `AuthTicketCookieHelper.identity()` to return not only the `Identity` (as before) but also the corresponding valid `AuthTicket` that was found. When verifying an HTML auth cookie (with `helper.identity(html_authcookie, request)`), `CookiePolicy` also needs to get the `AuthTicket` so that it can re-use the same ticket it has to issue the browser with a new API auth cookie to go with a lone HTML one.